packages/inter-protocol/src/price/fluxAggregator.contract.js

```diff
@@ -54,10 +67,53 @@ export const start = async (zcf, privateArgs, baggage) => {
     storageNode,
     marshaller,
   );
+  trace('got fa', fa);
+
+  const { augmentPublicFacet, makeGovernorFacet } = await handleParamGovernance(
+    // @ts-expect-error FIXME include Governance params
+    zcf,
+    initialPoserInvitation,
+    {
+      // No governed parameters. Governance just for API methods.
+    },
+    storageNode,
+    marshaller,
+  );
+
+  /**
+   * Initialize a new oracle and send an invitation to administer it.
+   *
+   * @param {string} addr
+   */
+  const addOracle = async addr => {
+    const invitation = await E(fa.creatorFacet).makeOracleInvitation(addr);
+    // XXX imported from 'proposals' path
+    await reserveThenDeposit(
+      `fluxAggregator oracle ${addr}`,
+      namesByAddressAdmin,
+      addr,
+      [invitation],
+    );
+    return `added ${addr}`;
+  };
+
+  const governedApis = {
+    /**
+     * Add the specified oracles. May partially fail, such that some oracles are added and others aren't.
+     *
+     * @param {string[]} oracleIds
+     * @returns {Promise<Array<PromiseSettledResult<string>>>}
+     */
+    addOracles: oracleIds => {
+      return Promise.allSettled(oracleIds.map(addOracle));
+    },
+  };
 
+  const governorFacet = makeGovernorFacet(fa.creatorFacet, governedApis);
   return harden({
-    creatorFacet: fa.creatorFacet,
-    publicFacet: fa.publicFacet,
+    creatorFacet: governorFacet,
+    // XXX this is a lot of API to put on every public facet
```

Comment:
You  can pick and choose if it shouldn't all be there.

Reply:
Oh good. I think this came when I ran into some type problems in https://github.com/Agoric/agoric-sdk/pull/6963 . Removed now.